skate-plugin/src/test/kotlin/com/slack/sgp/intellij/ChangeLogParserTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij
+
+import com.google.common.truth.Truth.assertThat
+import java.time.LocalDate
+import org.junit.Test
+
+class ChangeLogParserTest {
```

Comment:
Nit for all of these: I think it's redundant to put "test" in the function name because they're all tests implicitly

Reply:
Got it: 06f3b2a13a569913c74020cb88ebd3897c0df6b8